migrator.go

```diff
@@ -56,6 +56,16 @@ type Index interface {
 	Option() string
 }
 
+// TableType table type interface
+type TableType interface {
```

Comment:
Hi @maiqingqiang 

Maybe we can start with the `Schema`, `Name`, `Type`, and `Comment` fields, which are less confusing for all drivers.